src/components/nav-bar/ProfileCard.tsx

```diff
@@ -17,7 +17,7 @@ export default function ProfileCard({ user }: { user: User | null }) {
   return (
     <Popover open={open} onOpenChange={setOpen}>
       <PopoverTrigger className={cn("flex items-center justify-center space-x-3 rounded-full")}>
-        <UserAvatar user={user} />
+        <UserAvatar imageUrl={user?.user_metadata?.avatar_url as string} />
```

Comment:
since it's nullable, let's add a fallback

Reply:
nvm noticed it's handled in `UserAvatar` so let's remove the `as string`

Reply:
I think the issue is `UserMetadata` is this:
```ts
export interface UserMetadata {
  [key: string]: any
}
```
So we don't know if `avatar_url` even exists. Is there a better way to help ts recognise this?